packages/warriorjs-cli/templates/readme/ability.ejs

```diff
@@ -1 +1 @@
-- `warrior.<%- ability.name %>()`: <%- ability.description -%>
+- `warrior.<%- ability.name %>(<%- ability.argumentsDescription %>)`: <%- ability.description -%>
```

Comment:
We need to update this to use the new `params` (let's not show the type information here, just parameter name and default value if it exists). Examples:

```
warrior.directionOf(space)
```

```
warrior.walk(direction = 'forward')
```